rollbar-spring-boot-webmvc/src/main/java/com/rollbar/spring/boot/webmvc/RollbarSpringBootExceptionResolver.java

```diff
@@ -0,0 +1,35 @@
+package com.rollbar.spring.boot.webmvc;
+
+import com.rollbar.notifier.Rollbar;
+import com.rollbar.spring.webmvc.RollbarHandlerExceptionResolver;
+import com.rollbar.web.listener.RollbarRequestListener;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.web.servlet.ServletListenerRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+@Order(Ordered.HIGHEST_PRECEDENCE)
+@ControllerAdvice
+public class RollbarSpringBootExceptionResolver extends RollbarHandlerExceptionResolver {
```

Comment:
We don't need to implement a `HandlerExceptionResolver` we can just use the one that we have packaged in `rollbar-spring-webmvc`. 
What we need to do for Spring boot is to make it aware about a `ServletContextInitializer` implementation, so we can register the `RollbarRequestListener` there, because seems that with Spring boot the `WebApplicationInitializer` is not called.

Reply:
Ok i refactored this in the next commit

Reply:
Btw, as part of this I removed the unit-test for RollbarSpringBootExceptionResolver. I don't think the RollbarSpringBootConfiguration class I added has much to test. It just registers a bean, and adds the RollbarRequestListener.class to the servlet listener.